internal/mtail/mtail.go

```diff
@@ -104,6 +106,33 @@ func (m *Server) StartTailing() error {
 	return nil
 }
 
+// 定时扫描通配符目录地址，对通配符地址扫描追加
+func (m *Server) TimingTailing() error {
+	// 判断地址中是否含有通配符
+	hasMeta := func(path string) bool {
+		magicChars := `*?[`
+		if runtime.GOOS != "windows" {
+			magicChars = `*?[\`
+		}
+		return strings.ContainsAny(path, magicChars)
+	}
+	for _, pattern := range m.logPathPatterns {
+		absPath, err := filepath.Abs(pattern)
+		if err != nil {
+			return err
+		}
+		d := filepath.Dir(absPath)
+		if !hasMeta(d) {
+			continue
+		}
+		glog.V(1).Infof("TimingTailing Tail pattern %q", pattern)
```

Comment:
Rename TimingTailing or remove because the log line will have the line number in it.